### DISCUSSION TITLE
Is it possible to apply textures on circles, quads?

### DISCUSSION THREAD

**Discussion by VityaSchel** [AllenDang/giu#519] [Q&A]
I tried solution described here https://github.com/ocornut/imgui/issues/497#issuecomment-173369690 to create rotating image widget but I can't find method to apply texture on quads. Is there a way to "push" and "pop" textures in draw list?

**Comment by gucio321** [comment]
Maybe giu.canvas is what you are looking for?

**Reply by VityaSchel** [reply]
> Maybe giu.canvas is what you are looking for?

Yes, I'm using canvas and want to fill circle with texture as it possible in imgui with PushTexture and PopTexture
